app/models/place.rb

```diff
@@ -181,6 +140,16 @@ def override_lat_lng?
     override_lat.present? && override_lng.present?
   end
 
+  def as_json(_options)
+    super(except: :id).merge(location: location_to_hash, _id: { "$oid": "dummy-oid" })
```

Comment:
`dummy-oid` smells a bit hacky... Is that something we could clean up? If not, could we have a comment explaining why it's needed?

Reply:
Ha, this is definitely a hack, I should probably have put a TODO for myself here. It's to keep the pact test working, but in fact I _think_ (just verifying now) that none of the consumers of this interface use this attribute (why would they care about a mongo internal id?), so I'll maybe make a mongoid version that doesn't pass it along either, update the pact to reflect it, then we're all good.

Reply:
Update:
-  I've removed the OID from the pact test expectation in gds-api-adapters, and altered the current main (Mongo) branch of imminence to get rid of it there. That frees us up to remove it here as well.
- Prompted by @1pretz1 's observation that overwriting as_json has a high chance of catching another developer out down the road, I've replaced this as_json with an as_safe_hash method and allowed the relevant controller to call that when doing a respond_to json.